client/src/components/auth/RegistrationForm.js

```diff
@@ -2,21 +2,29 @@
  * This file exports a Registration Form component used to display registration input.
  */
 
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import Jumbotron from 'react-bootstrap/Jumbotron';
 import Button from 'react-bootstrap/Button';
 import Form from 'react-bootstrap/Form';
 import { Col } from 'react-bootstrap';
 import { createTeam } from '../../actions/team';
 import './Register.css';
```

Comment:
Example that Register is used as naming convention, file name is also RegistrationForm.